### PR TITLE
[BrowserKit] Added isFollowingRedirects and getMaxRedirects methods

### PR DESCRIPTION
**Pull request by Naktibalda**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | yes |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets |  |
| License | MIT |
| Doc PR |  |

### CODE REVIEW

**Comment by fabpot** [comment]
What is your use case?

**Comment by Naktibalda** [comment]
https://github.com/Codeception/Codeception/pull/2329

I want to intercept the redirect, do some actions and then follow the redirect.
My problem is that the instance of BrowserKit Client is public and following redirects can be disabled or limited by 3rd party code and I need to know that.

File: src/Symfony/Component/BrowserKit/Client.php [hunk 1]
```diff
@@ -80,6 +80,16 @@ public function followRedirects($followRedirect = true)
     }
 
     /**
+     * Returns whether client automatically follow redirects or not.
+     *
+     * @return bool
+     */
+    public function getFollowRedirects()
```
Comment (sstok) [inline src/Symfony/Component/BrowserKit/Client.php]: As this a boolean value it should be something like `is[...]()`.
`isRedirectFollowed()`

**Comment by fabpot** [comment]
Thank you @Naktibalda.

**Pull request closed without merging**